param/__init__.py

```diff
@@ -1149,10 +1149,14 @@ def get_range(self):
         raise NotImplementedError("get_range() must be implemented in subclasses.")
 
 
-class ObjectSelector(SelectorBase):
+class Selector(SelectorBase):
     """
     Parameter whose value must be one object from a list of possible objects.
 
+    By default, if no default is specfied, picks the first object from
```

Comment:
Typo: `specfied -> specified`. I'll make a new PR with these minor fixes now this has been merged.

Reply:
Fixed in #507